packages/react/src/generators/application/lib/normalize-options.ts

```diff
@@ -95,10 +95,6 @@ export async function normalizeOptions<T extends Schema = Schema>(
 
   assertValidStyle(options.style);
 
-  if (options.bundler === 'vite' && options.unitTestRunner !== 'none') {
```

Comment:
i search the code, the same line is also in packages/web/src/generators/application/application.ts
should web to be changed as well?